carma-messenger-config/chevrolet_tahoe_2018/.env

```diff
@@ -1,2 +1,2 @@
-DOCKER_ORG=usdotfhwastoldev
-DOCKER_TAG=develop
\ No newline at end of file
+DOCKER_ORG=usdotfhwastol
+DOCKER_TAG=carma-system-4.5.0
```

Comment:
This should indicate the DOCKER_ORG or DOCKER_TAG that is in test or release. So it should be `usdotfhwastolcandidate` and `powler`

Reply:
On the other hand, if some previous images need to be used (like carma-system-4.5.0) they should be hardcoded in the docker-compose.yml itself. This is how we have been doing for all xil_carma releases.

Reply:
I understand it is kind of doing the same thing with another way, but this way, when we switch to master, we only need to change here instead of multiple places in docker-comppose.yml as well.

Reply:
updated